tests/test_mixins.py

```diff
@@ -182,3 +164,52 @@
     program_runner.add(_prog)
     results = await program_runner.join()
     assert len(results) == n
+
+
+@mark.asyncio
+async def test_invert_share(test_preprocessing):
+    n, t = 3, 1
+    test_preprocessing.generate("triples", n, t)
+    test_preprocessing.generate("rands", n, t)
+
+    async def _prog(context):
+        share = test_preprocessing.elements.get_rand(context)
+        # assert 1/x * x == 1
+        assert await(await(await(Inverter.invert_share(context, share))
```

Comment:
We need to fix this. This is crazy. This needs to align with the data flow design. Lets address this after this PR.

Reply:
Agreed-- it was a massive PITA to figure out how to get this to compile and all